scripts/ci/jobs/gke_upgrade_tests.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env -S python3 -u
 
+# TODO(sbostick): just tracking relevant files
+
 """
 Run the upgrade test in a GKE cluster
 """
```

Comment:
You probably need:
```
os.environ["ORCHESTRATOR_FLAVOR"] = "k8s"
os.environ["SENSOR_HELM_DEPLOY"] = "true"
```
at line 15. `k8s` because central is not openshift 🤞

Reply:
Got it, trying now...